app/whats-new/index.js

```diff
@@ -5,12 +5,12 @@ const whatsNewContentService = require('../../common/services/whats-new-content'
 
 // Define routes
 router.get('/', async (req, res, next) => {
```

Comment:
```suggestion
router.get('/', async (req, res) => {
```

Since we don't use this. I'm also wondering if this is why Express wasn't catching the error for us...

Reply:
`next` is used at line 13

Reply:
Oh yes, that was a poorly worded comment.

I was still trying to figure out why we needed to call `next` explicitly and why Express wasn't catching the error for us (as this is something we do in numerous other places) and wondered if this could be the reason.

Reply:
Ah, I see what you mean now. @georgeschena what happens without the try/catch?